docs/sources/developers/plugins/backend/plugin-protocol.md

```diff
@@ -11,24 +11,34 @@ title: Plugin protocol
+
+{{% admonition type="note" %}}
+The plugin protocol lives in the [Grafana Plugin SDK for Go]({{< relref "grafana-plugin-sdk-for-go/" >}}) because Grafana itself uses parts of the SDK as a dependency.
+{{% /admonition %}}
 
 ## Versioning
 
-Additions of services, messages and fields in the latest version of the plugin protocol are expected to happen, but should not introduce any breaking changes. If breaking changes to the plugin protocol is needed, a new major version of the plugin protocol will be created and released together with a new major Grafana release. Grafana will then support both the old and the new plugin protocol for some time to make sure existing backend plugins continue to work.
+From time to time, Grafana will offer additions of services, messages, and fields in the latest version of the plugin protocol. We don't expect these updates to introduce any breaking changes. However, if we must introduce breaking changes to the plugin protocol, then we'll create a new major version of the plugin protocol.
+
+Grafana will release new major versions of the plugin protocol alongside new major Grafana releases. When this happens, we'll then support both the old and the new plugin protocol for some time to make sure existing backend plugins continue to work.
 
-Because Grafana maintains the plugin protocol, the plugin protocol attempts to follow Grafana's versioning, However, that doesn't automatically mean that a new major version of the plugin protocol is created when a new major release of Grafana is released.
+The plugin protocol attempts to follow Grafana's versioning, However, that doesn't mean that we will automatically create a new major version of the plugin protocol when a new major release of Grafana is released.
 
 ## Writing plugins without Go
 
-If you want to write a backend plugin in another language than Go, then it’s possible as long as the language supports [gRPC](https://grpc.io/). However, writing a plugin in Go is recommended and has several advantages that should be carefully taken into account before proceeding:
+If you want to write a backend plugin in a language other than Go, then it's possible as long as the language supports gRPC. However, we recommend that you develop your plugin in Go for several reasons:
 
-- There's an official [SDK]({{< relref "grafana-plugin-sdk-for-go/" >}}) available.
-- Single binary as the compiled output.
-- Building and compiling for multiple platforms is easy.
-- A statically compiled binary (in most cases) doesn't require any additional dependencies installed on the target platform enabling it to run “everywhere”.
-- Small footprint in regards to binary size and resource usage.
+- We offer an official plugin SDK
+- The compiled output is a single binary
+- Writing for multiple platforms is easy. Typically, no additional dependencies must be installed on the target platform.
+- Small footprint for binary size
+- Small footprint for resource usage
```

Comment:
```suggestion
- We offer an official plugin SDK.
- The compiled output is a single binary.
- Writing for multiple platforms is easy. Typically, no additional dependencies must be installed on the target platform.
- Small footprint for binary size.
- Small footprint for resource usage.
```

Third item has a full stop because sentences.. so all have to have full stops.